### PR TITLE
EZSPA-364 Convert Livy Server to StatefulSet

### PR DESCRIPTION
**Pull request by idzikovskyi**

### CODE REVIEW

**Review by akshayharyani — DISMISSED** [review]
LGTM

File: charts/livy/livy-chart/templates/statefulset.yaml [hunk 1]
```diff
@@ -8,17 +8,12 @@ metadata:
   {{- end }}
   {{- end }}
 spec:
-  progressDeadlineSeconds: {{ .Values.progressDeadlineSeconds }}
   replicas: {{ .Values.replicaCount }}
   revisionHistoryLimit: {{ .Values.revisionHistoryLimit }}
```
Comment (apurwaj2) [inline charts/livy/livy-chart/templates/statefulset.yaml:12]: revisionHistoryLimit won't be needed for statefulset, right?

File: charts/livy/livy-chart/values.yaml [hunk 2]
```diff
@@ -40,8 +40,6 @@ restartPolicy: "Always"
 
 automountServiceAccountToken: true
 
-progressDeadlineSeconds: 600
-
 revisionHistoryLimit: 10
```
Comment (apurwaj2) [inline charts/livy/livy-chart/values.yaml:43]: Same as the above comment

**Review by apurwaj2 — DISMISSED** [review]
Other than the two comments, lgtm.

File: charts/livy/livy-chart/templates/statefulset.yaml (hunk 1 above)
Comment (idzikovskyi) [reply charts/livy/livy-chart/templates/statefulset.yaml]: Oh, yeah. Thanks for noticing this.

**Review by apurwaj2 — DISMISSED** [review]
LGTM

**Review by AlexNavara — APPROVED** [review]

**Pull request merged**